website/content/docs/concepts/index.mdx

```diff
@@ -49,14 +49,11 @@ supported for the specific inputs and examples with those inputs.
 
 # Next Steps
 
-Be sure Boundary is able to run locally with the instructions at [Getting
-Started](/docs/getting-started). Then, learn how to create targets and initiate
-a session with [Connect to Your First
-Target](/docs/getting-started/connect-to-target).
-
-# Further Reading
-
-For more information see our general recommendations for [deployment
-architecture](/docs/installing/high-availability), and see the [security
-model](/docs/concepts/security) documentation for an explanation of the security
-foundations of Boundary.
+Boundary can be deployed easily as a managed service using the HashiCorp Cloud
+Platform. Check the [Getting Started](/docs/getting-started) section to learn
+more about HCP Boundary.
+
+Alternatively, Boundary can be run locally in [Dev
```

Comment:
This makes it seem like there are _only_ two options -- HCP Boundary or Dev Mode. Really the two deployment options are HCP Boundary or local open-source, and dev mode is just a useful learning/testing tool.

Reply:
great callout. I added more context under the Next Steps section. Thoughts?

Reply:
Checked, 👍